.docker/wp-config.php

```diff
@@ -108,6 +108,11 @@
 define( 'WORDCAMP_QBO_CONSUMER_SECRET', '' );
```

Comment:
The edits on this file are making it fail the phpcs check – maybe add `// phpcs:ignoreFile` to the start of the file?

Reply:
Hrm, none of the errors are related to lines that I actually changed. Is there a way to limit the phpcs check to only changed lines? `// phpcs:ignoreFile` seems drastic.

Reply:
yeah, that was a todo item we decided to come back to– I've mostly been fixing issues as they come up so that the whole file passes, as a way to clean up as we go, but I figured wp-config would be a special case that could be ignored.

Reply:
There were only a few errors, so I just fixed the file: c1cbb35c